Code/Framework/AzToolsFramework/Tests/ActionManager/ToolBarManagerTests.cpp

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Tests/ActionManager/ActionManagerFixture.h>
+#include <AzToolsFramework/API/ToolsApplicationAPI.h>
+
+#include <QToolBar>
+
+namespace UnitTest
+{
+    TEST_F(ActionManagerFixture, RegisterToolBar)
+    {
+        auto outcome = m_toolBarManagerInterface->RegisterToolBar("o3de.toolbar.test", {});
+        EXPECT_TRUE(outcome.IsSuccess());
+    }
+
+    TEST_F(ActionManagerFixture, RegisterMenuToolBar)
```

Comment:
suggestion: This is likely a preference thing, but one thought I had when reading these test names is that they might be clearer as being the expected outcome, as opposed the action to perform. For example, this test would become `MenuToolBarIsRegistered` - it's pretty subtle but it slightly better describes the behavior and expected outcome rather than just the action (that potentially could fail). It's by no means critical, just a thought to make the tests even better than they are already 🙂

Reply:
I will keep this in mind and do a pass in a follow-up!